sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeFormatterHelper.scala

```diff
@@ -244,6 +250,12 @@ private object DateTimeFormatterHelper {
           for (style <- unsupportedNarrowTextStyle if patternPart.contains(style)) {
             throw new IllegalArgumentException(s"Too many pattern letters: ${style.head}")
           }
+          if (bugInStandAloneForm && (patternPart.contains("LLL") || patternPart.contains("qqq"))) {
+            throw new IllegalArgumentException("The current JDK has a bug to support stand-alone " +
```

Comment:
Maybe, `JDK` -> `Java` because there is JRE environment, too?

Reply:
This is a bug in the time library of JDK, and is not related to JRE. I think JDK is more precise.

Reply:
What do you mean by `not related to JRE`? `DateTimeFormatter.ofPattern("LLL", Locale.US)` is working correctly on JRE or `DateTimeFormatter.ofPattern("LLL", Locale.US)` is absent in JRE?

Reply:
Maybe we have different interpretation of JDK and JRE.

This is a bug in the `java.time` library, not in JVM. I'm not sure how to refer to it.

Reply:
I think 'Java' is more relevant term here since I guess that this error occurs on `java` command in both JDK and JRE. Both use Java class library (JCL).   
Here are definitions of both terms at https://java.com/en/download/faq/techinfo.xml

Reply:
I got your intention. I wrote the previous one before I read [your message](https://github.com/apache/spark/pull/28646#discussion_r430898129).

How about the term Java class library (JCL)?
https://en.wikipedia.org/wiki/Java_Class_Library

Reply:
JCL is not a widely used word. I picked `Java` to make it general and easy to understand.

Reply:
I see. It is familiar to me (and Kris) :). Got general feeling. No objection.